cleanify.py

```diff
@@ -60,7 +62,7 @@ def save_decoded_image(img, name):
 # define transforms
 transform = transforms.Compose([
     transforms.ToPILImage(),
-    transforms.Resize((224, 224)),
+    transforms.Resize((64, 64)),
```

Comment:
We can probably get rid of this and the PIL conversion since our input is 64x64.